src/exporters/STLExporter.ts

```diff
@@ -131,39 +148,39 @@ STLExporter.prototype = {
       }
     }
 
-    function writeNormal(vA, vB, vC) {
+    function writeNormal(vA: Vector3, vB: Vector3, vC: Vector3): void {
       cb.subVectors(vC, vB)
       ab.subVectors(vA, vB)
       cb.cross(ab).normalize()
 
       normal.copy(cb).normalize()
 
-      if (binary === true) {
+      if (binary === true && output instanceof DataView) {
         output.setFloat32(offset, normal.x, true)
         offset += 4
         output.setFloat32(offset, normal.y, true)
         offset += 4
         output.setFloat32(offset, normal.z, true)
         offset += 4
       } else {
-        output += `\tfacet normal ${normal.x} ${normal.y} ${normal.z}\n`
+        output += '\tfacet normal ' + normal.x + ' ' + normal.y + ' ' + normal.z + '\n'
```

Comment:
is there a reason you decided to not use the template literal? I think it's easier to read, no?

Reply:
Yes, when I copied the `STLExporter` implementation of `threejs`, I too wondered why this part was rewritten with `+` operator.
I also think it's easier to read if it's written in template literal, so I'll fix it.